src/main/kotlin/dev/anvith/alphabet/GenerateIntentionColor.kt

```diff
@@ -22,29 +26,30 @@ class GenerateIntentionColor : PsiElementBaseIntentionAction(), IntentionAction
         return NAME
     }
 
-
     override fun isAvailable(project: Project, editor: Editor?, element: PsiElement): Boolean {
         return element is XmlToken && element.parents.any { it is XmlTag && it.name == COLOR_TAG && it.parentTag?.name == RES_TAG }
     }
 
     override fun invoke(project: Project, editor: Editor?, element: PsiElement) {
         val colorTag = element.parents.first { it is XmlTag && it.name == COLOR_TAG }
         val percentageString = Messages.showInputDialog(project, PERCENTAGE, NAME, null)
-
+        val percentages = percentageString.orEmpty().trim().split(",")
```

Comment:
Consider trimming after mapping
->`val percentages = percentageString.orEmpty().split(",").map{it.trim()}`
this is to ensure values like
`10, 12, 38 ` are handled gracefully rather than just `10,12,38 `

Reply:
Rest looks good.